src/main/kotlin/near/vanilla/death/database/DeathListener.kt

```diff
@@ -0,0 +1,17 @@
+package near.vanilla.death.database
+
+import org.bukkit.entity.Player
+import org.bukkit.event.EventHandler
+import org.bukkit.event.EventPriority
+import org.bukkit.event.Listener
+import org.bukkit.event.entity.PlayerDeathEvent
+
+object DeathListener : Listener {
+
+    @EventHandler(priority=EventPriority.MONITOR)
+    fun onDeath(deathEvent: PlayerDeathEvent) {
+        info("Death event received for player: ${deathEvent.entity.displayName} with uuid: ${(deathEvent.entity as Player).uniqueId.toString()}")
+        DeathDatabase[deathEvent.entity] = deathEvent.drops
+    }

```

Comment:
Not sure what'd be the best here. I think ideally, we should:
1. Make sure that event wasn't cancelled (either by an appropriate annotation parameter on method or explicit check)
2. Store all of player inventory contents, instead of drops (for example plugins might modify drops adding heads or removing some items with Vanishing enchantment etc.)